crates/router/src/connector/adyen/transformers.rs

```diff
@@ -1666,20 +1706,21 @@ pub fn get_adyen_response(
 > {
     let status =
         storage_enums::AttemptStatus::foreign_from((is_capture_manual, response.result_code));
-    let error = if response.refusal_reason.is_some() || response.refusal_reason_code.is_some() {
-        Some(types::ErrorResponse {
-            code: response
-                .refusal_reason_code
-                .unwrap_or_else(|| consts::NO_ERROR_CODE.to_string()),
-            message: response
-                .refusal_reason
-                .unwrap_or_else(|| consts::NO_ERROR_MESSAGE.to_string()),
-            reason: None,
-            status_code,
-        })
-    } else {
-        None
-    };
+    let error: Option<types::ErrorResponse> =
+        if response.refusal_reason.is_some() || response.refusal_reason_code.is_some() {
+            Some(types::ErrorResponse {
+                code: response
+                    .refusal_reason_code
+                    .unwrap_or_else(|| consts::NO_ERROR_CODE.to_string()),
+                message: response
+                    .refusal_reason
+                    .unwrap_or_else(|| consts::NO_ERROR_MESSAGE.to_string()),
+                reason: None,
+                status_code,
+            })
+        } else {
+            None
+        };
```

Comment:
Do we need this change, everything looks similar only type introduced

Reply:
I will remove the type